src/applications/vaos/services/location/transformers.js

```diff
@@ -356,3 +331,58 @@ export function transformCommunityProviders(providers) {
     };
   });
 }
+
+export function transformSettings([request = [], direct = []]) {
```

Comment:
This code is somewhat complicated because I didn't want to assume that we'd get the same facilities in the request and direct arrays, nor that each facility in those arrays would have the same list of types of care, because VATS data has sometimes been strange.